modules/fx/arrow-fx/src/main/kotlin/arrow/fx/extensions/schedule.kt

```diff
@@ -43,45 +43,39 @@ interface ScheduleApplicative<F, Input> : Applicative<SchedulePartialOf<F, Input
   fun MF(): Monad<F>
 
   override fun <A> just(a: A): Kind<SchedulePartialOf<F, Input>, A> =
-    Schedule.forever<F, A>(MF()).const(a) as Schedule<F, Input, A>
+    Schedule.once<F, A>(MF()).const(a) as Schedule<F, Input, A>
```

Comment:
This should break applicative laws. Schedule has an applicative instance with `const . forever` as `just` and `and` as `ap`.
`and` combines two schedules by combining their decision with `and` for the decision to continue and `max` for the delay. This means that the neutral element produced by `just` must always continue and have the minimum delay, which is why it used forever.

Reply:
I see what you mean and makes sense. I believe the tests passed anyway so I'll try to check them out as well 👍